### ISSUE TITLE
Get Started With Tensorflow

### ISSUE THREAD

**Issue by Praneet460** [pydatadelhi/talks#86]
**Get Started with TensorFlow**
- Abstract (2-3 lines)
Cover the topics to help students get started with writing their own machine learning programs.

- Brief Description and Contents to be covered
Classification and Regression Algorithms as well as Deep Neural Network

- Pre-requisites for the talk
Little bit knowledge about Python is enough.

- Time required for the talk
1 hr.
- Link to slides
[Click Here](https://github.com/Praneet460/MLCC)

- Will you be doing hands-on demo as well?
Yes

- Link to ipython notebook (if any)
[Click Here](https://github.com/Praneet460/MLCC)

- About yourself
Enthusiastic about machine learning and data science.
Contact - nigampraneet460@gmail.com

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Yes

- Any query ?

**Comment by shagunsodhani** [comment]
Hey @Praneet460 Thank you for your proposal. Could you please outline what aspects you are planning to cover (I am sure you are not planning to cover the entire repo in a one-hour setting). I think it will be really interesting if you mention the tf2.0 API as well - as that will be the new standard.

**Comment by Praneet460** [comment]
I will cover the tensorflow basics - like,  how to create your first neural network.  And tf.keras api.  All IntroToTF parts

**Comment by shagunsodhani** [comment]
Cool :) Do mention Tf2.0. Best of luck!

**Comment by Praneet460** [comment]
Yes,  sure TF 2.0 comes up with a lot of new features.  I would love to introduce them to the audience.
